helpers/query.go

```diff
@@ -0,0 +1,90 @@
+package helpers
+
+import (
+	"strconv"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	tmclient "github.com/cosmos/cosmos-sdk/x/ibc/light-clients/07-tendermint/types"
+	"github.com/cosmos/relayer/relayer"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+)
+
+// QueryBalance is a helper function for query balance
+func QueryBalance(chain *relayer.Chain, address string, showDenoms bool) (sdk.Coins, error) {
+	coins, err := chain.QueryBalanceWithAddress(address)
+	if err != nil {
+		return nil, err
+	}
+
+	if showDenoms {
+		return coins, nil
+	}
+
+	h, err := chain.QueryLatestHeight()
+	if err != nil {
+		return nil, err
+	}
+
+	dts, err := chain.QueryDenomTraces(0, 1000, h)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(dts.DenomTraces) == 0 {
+		return coins, nil
+	}
+
+	var out sdk.Coins
+	for _, c := range coins {
+		for _, d := range dts.DenomTraces {
+			switch {
+			case c.Amount.Equal(sdk.NewInt(0)):
+			case c.Denom == d.IBCDenom():
+				out = append(out, sdk.NewCoin(d.GetFullDenomPath(), c.Amount))
```

Comment:
```suggestion
				out = append(out, sdk.NewCoin(d.IBCDenom(), c.Amount))
```
this will panic if provided the full denom path